server/routes/prisonerList/prisonerListView.ts

```diff
@@ -29,31 +29,42 @@ export default class PrisonerListView {
       sortBy: this.sortBy,
       sortOrder: this.sortOrder,
       renderPaginationControls: this.pagedPrisonerSearchSummary.totalPages > 1,
-      items: buildItemsArray(this.pagedPrisonerSearchSummary, this.searchTerm, this.statusFilter),
+      items: buildItemsArray({
+        pagedPrisonerSearchSummary: this.pagedPrisonerSearchSummary,
+        searchTerm: this.searchTerm,
+        statusFilter: this.statusFilter,
+        sortBy: this.sortBy,
+        sortOrder: this.sortOrder,
+      }),
       results: buildResults(this.pagedPrisonerSearchSummary),
       previousPage: getPreviousPage(this.pagedPrisonerSearchSummary),
       nextPage: getNextPage(this.pagedPrisonerSearchSummary),
     }
   }
 }
 
-const buildItemsArray = (
-  pagedPrisonerSearchSummary: PagedPrisonerSearchSummary,
-  searchTerm: string,
-  statusFilter: string,
-): Item[] => {
+const buildItemsArray = (config: {
```

Comment:
I've changed this to a map of config options, as the individual args were getting a bit long and messy 👍